CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ else( PULSE_AUDIO_SUPPORT )
 endif( PULSE_AUDIO_SUPPORT )
 
 if( NOT MSVC )
-	add_definitions( "-std=gnu99" )
+	add_definitions( "-std=gnu11" )
```

Comment:
@EliasOenal This PR bumps the standard to gnu11 due to usage of `timespec_get` to get higher precision timestamps. Will that be a problem? Its still gnu, but included more recent apis.

Reply:
Switching to c11/gnu11 is fine with me. I believe most systems support it these days. 😀